static/specification.html

```diff
@@ -661,9 +662,19 @@ <h2>
           <dfn>amount</dfn> attribute
         </h2>
         <p>
-          The amount delivered by an ILP packet. When getting, returns the
-          value it was initialized with.
+          The amount <em>sent</em>. When getting, returns the value it was
+          initialized with.
         </p>
+        <div class="warning">
+          <p>
+            The `amount` attribute previously reflected the amount
+            <em>received</em>.
```

Comment:
So by using sentAmount it means you could do things like an http request to initiate a payment on the backend, and rather than have to wait for the STREAM to finish, the backend could send to the incoming payment and respond immediately ?

Reply:
Yes

Reply:
Should there be any limitations on the amount of different assets "sent" in a given session ?  Just 1? 
Theoretically a provider could send using multiple source currencies, and it would complicate things a little having to check for that.  

Given we are going to use "monetization" event for tips, how does one know when a tip is sent? Once a value threshold has been crossed?

Reply:
@adrianhopebailie 

Any thoughts here?